pkg/machines/components/create-vm-dialog/Password.jsx

```diff
@@ -0,0 +1,125 @@
+import React from "react";
```

Comment:
This will became the only file pkg/machine which starts with upper case. We need get our file naming practices in order :D . I would prefer it to be lowercase.

Reply:
Renamed.